app/services/employee_importer.rb

```diff
@@ -0,0 +1,74 @@
+require "slack-ruby-client"
+
+class EmployeeImporter
+  def initialize
+    configure_slack
+  end
+
+  def import(dry_run = false)
+    import_results = {created: 0, skipped: 0, dry_run: false}
+    slack_user_finder = SlackUserFinder.new("", client)
+    total_employees = slack_user_finder.users_list.count
+
+    slack_user_finder.users_list.each_with_index do |user, index|
+      if !dry_run
```

Comment:
might be slightly more readable to have the conditional logic start with`if dry_run`